.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: java
+
+jdk: 
+- oraclejdk8
+
+before_install:
+- git clone https://github.com/inf295uci-2015/primitive-hamcrest.git  
+- cd primitive-hamcrest  
+- mvn install
+- cd ..
+- git clone https://github.com/spideruci/tacoco.git
+- cd tacoco
+- mvn install
+- cd ..
+
+after_success:
+- mvn clean test -P full
```

Comment:
what happens when you do a `mvn clean verify -P full`?

Reply:
That should work haven't tried it out yet. What is the difference between the verify and test goal? I see it also runs blinky-diagnostics, but does it also run all the unit tests? I quickly went through it manually and I think it does right? So shall I change test to verify?